browser-extension/mv2/src/client/js/requestResponseRuleHandler.js

```diff
@@ -103,9 +103,33 @@ RQ.RequestResponseRuleHandler.cacheResponseRules = async () => {
   );
 };
 
```

Comment:
@nafees87n Right now, we are adding all the code to requestResponseRuleHandler, but the new code is related to Redirect Rules, which makes it semantically incorrect. 

Can we log an additional task to refactor the codebase and instead of calling requestResponseRuleHandler, maybe we can rename these files/code to ajaxRequestsHandler

Reply:
Sure